src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -494,19 +494,21 @@ def nlcd_kfactor(result):
     return output
 
 
-def multi_mapshed(aoi, wkaoi):
+def multi_mapshed(aoi, wkaoi, layer_overrides={}):
```

Comment:
Perhaps I should replace this with `dict`

```suggestion
def multi_mapshed(aoi, wkaoi, layer_overrides=dict):
```

Reply:
Yes!